### ISSUE TITLE
test project crash in android 5.0 device (Galaxy Note3) when use 3.12

### ISSUE THREAD

**Issue by pbs0512** [cocos2d/cocos2d-x#16251]
- cocos2d-x version: 3.12
- devices test on: Galaxy Note3(android 5.0)
- developing environments
  - NDK version: 11c

Steps to Reproduce:
1. cocos2d-x\tests\cpp-tests>cocos run -p android
2. run and crash

```
********** Crash dump: **********
Build fingerprint: 'samsung/hlteskt/hlteskt:5.0/LRX21V/N900SKSU0GPC5:user/release-keys'
pid: 18135, tid: 18273, name: GLThread 14604  >>> org.cocos2dx.cpp_tests <<<
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x1a
Stack frame D/CrashAnrDetector(  821):     #00 pc 01c54f08  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::Label::computeHorizontalKernings(std::basic_string<char16_t, std::char_traits<char16_t>, std::allocator<char16_t> > const&)+144)
Stack frame D/CrashAnrDetector(  821):     #01 pc 01c56fcc  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::Label::updateContent()+392)
Stack frame D/CrashAnrDetector(  821):     #02 pc 01c5a004  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::Label::getContentSize() const+64)
Stack frame D/CrashAnrDetector(  821):     #03 pc 01c82f38  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::MenuItemLabel::setLabel(cocos2d::Node*)+124)
Stack frame D/CrashAnrDetector(  821):     #04 pc 01c832fc  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::MenuItemLabel::initWithLabel(cocos2d::Node*, std::function<void (cocos2d::Ref*)> const&)+188)
Stack frame D/CrashAnrDetector(  821):     #05 pc 01c83408  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::MenuItemLabel::create(cocos2d::Node*, std::function<void (cocos2d::Ref*)> const&)+156)
Stack frame D/CrashAnrDetector(  821):     #06 pc 0118b50c  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (TestList::runThisTest()+1948)
Stack frame D/CrashAnrDetector(  821):     #07 pc 017589d0  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (TestController::TestController()+280)
Stack frame D/CrashAnrDetector(  821):     #08 pc 0175ec18  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (TestController::getInstance()+140)
Stack frame D/CrashAnrDetector(  821):     #09 pc 01189440  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (AppDelegate::applicationDidFinishLaunching()+1716)
Stack frame D/CrashAnrDetector(  821):     #10 pc 0177664c  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (cocos2d::Application::run()+44)
Stack frame D/CrashAnrDetector(  821):     #11 pc 0177b1a8  /mnt/asec/org.cocos2dx.cpp_tests-1/lib/arm/libcpp_tests.so (Java_org_cocos2dx_lib_Cocos2dxRenderer_nativeInit+256)
Stack frame D/CrashAnrDetector(  821):     #12 pc 00020cb1  /data/dalvik-cache/arm/mnt@asec@org.cocos2dx.cpp_tests-1@base.apk@classes.dex
```

----------------------- test 1
1. change CCFontAtlas.cpp  https://github.com/cocos2d/cocos2d-x/commit/66fac82f007514e9ec4021515e325c49d129a45b
2. but crash while launch

----------------------- test 2
1. cocos run -p android --app-abi=armeabi-v7a
2. ok

**Comment by alfogrillo** [comment]
It was fixed here #16169 
Take a look!

**Comment by pbs0512** [comment]
@Drakon-Cocos thank you
